powpeg-details.js

```diff
@@ -31,14 +31,25 @@ module.exports = async (web3, networkSettings) => {
     const federationCreationBlockNumber = await bridge.methods.getFederationCreationBlockNumber().call();
 
     let redeemScript;
-    if (federationCreationBlockNumber >= networkSettings.getNetworkUpgradesActivationHeights().getActivationHeight('iris')) {
-        redeemScript = RedeemScriptParser.getErpRedeemScript(
-            btcPublicKeys, 
-            networkSettings.getErpDetails().getErpPublicKeys(), 
-            networkSettings.getErpDetails().getCsvValue()
-        ).toString('hex');
-    } else {
+    if (federationCreationBlockNumber < networkSettings.getNetworkUpgradesActivationHeights().getActivationHeight('iris')) {
```

Comment:
I think this could be simpler if we start checking by the latest HF

```
if (federationCreationBlockNumber > hopActivation) {
    return bridge.getRedeemScript();
}
if (federationCreationBlockNumber > irisActivation) {
    ....
}

and so on
```

Reply:
Checking From The Latest HF Might Be Hard To Test.. 
For Example The federationCreationBlockNumber For Each HF Is As Follows: hop - 1500 , iris - 1000, rest - 500.. If 500 Is Pass As The federationCreationBlockNumber, The First Condition Kicks In , Meanwhile We Need It To Work For The Last Condition/ Least HF